help/en/manual/DecoderPro/VSD_Manager.shtml

```diff
@@ -128,6 +128,13 @@
 
       <p>Sounds can also be activated by a throttle assigned to the
       same address as the VSDecoder.</p>
+	  <p>And similar the "Engine Start" button 
+      <a href="VSD_File_and_Config.shtml#engine-startstop">can be
+      activated</a> by a throttle key, if desired.</p>
+      <p><em>Note 1:</em> To start the Engine the throttle speed must be 0.<br>
+      <em>Note 2:</em> Diesel engines normally have a starting-sound. 
+      Throttle changes will not be processed util the starting-sound
```

Comment:
Apologies, but just noticed a typo - 'util' should be 'until'

Reply:
@mattharris Thanks again. It seams that I have a kind of talent in such things.